src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Tokenizes the user input so the app can parse it. 
+ * 
+ * @author (Sarah Abowitz) 
+ * @version (V1.0.0 | 5.20.17)
+ */
+
+package codeu.chat.util;
+
+import java.io.IOException;
+import java.lang.StringBuilder;
+
+public final class Tokenizer {
```

Comment:
Nice job with the javadoc on this class and its methods! FYI, generally it's not necessary to add javadoc to private methods.